### ISSUE TITLE
ERROR:pgsync.base: Exception (psycopg2.errors.UndefinedTable) relation "_pkey_view" does not exist

### ISSUE THREAD

**Issue by pushkar-anand** [toluaina/pgsync#131]
PGSync version: 2.0.0

Postgres version: 12.7

Elasticsearch version: 7.7

Redis version: 6.0.9

Python version: 3.7.10

Problem Description: 

The bootstrap step is failing. Here is my `schema.json`

```
[
    {
        "database": "db",
        "index": "pgsync-test",
        "mappings": {...},
        "nodes": {
            "table": "table",
            "schema": "schema",
            "columns":
            [
                "col1", "col2"
            ]
        }
    }
]
```

Error Message (if any): 

```
2021-06-07 17:31:01.390:ERROR:pgsync.base: Exception (psycopg2.errors.UndefinedTable) relation "_pkey_view" does not exist
LINE 2: FROM _pkey_view
             ^

[SQL: SELECT table_name, primary_keys
FROM _pkey_view]
(Background on this error at: http://sqlalche.me/e/14/f405)
Traceback (most recent call last):
  File "/Users/pushkar/Projects/pgsync-test/venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1707, in _execute_context
    cursor, statement, parameters, context
  File "/Users/pushkar/Projects/pgsync-test/venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 717, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedTable: relation "_pkey_view" does not exist
LINE 2: FROM _pkey_view
             ^

```

**Comment by timbarkerSE** [comment]
If you've previously run the bootstrap step and it has failed, you may need to run it with the --teardown to reset everything then try again.

**Comment by pushkar-anand** [comment]
@timbarkerSE The teardown step worked, but when I run the bootstrap again after the teardown I get another error:
```
Traceback (most recent call last):
  File "/Users/pushkar/Projects/pgsync-test/venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1707, in _execute_context
    cursor, statement, parameters, context
  File "/Users/pushkar/Projects/pgsync-test/venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 717, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.SyntaxError: syntax error at or near ")"
LINE 2: FROM (VALUES ) AS v (table_name, foreign_keys)
```

**Comment by toluaina** [comment]
Sorry I'm late addressing this. 
Just published version 2.10 off master to address this now.
